pkg/rescheduler/evictions/evictions.go

```diff
@@ -49,7 +49,7 @@ func EvictPod(client clientset.Interface, pod *v1.Pod, policyGroupVersion string
 	} else if apierrors.IsTooManyRequests(err) {
 		return false, fmt.Errorf("error when evicting pod (ignoring) %q: %v", pod.Name, err)
 	} else if apierrors.IsNotFound(err) {
-		return true, fmt.Errorf("pod not found when evicting %q: %v", pod.Name, err)
+		return false, fmt.Errorf("pod not found when evicting %q: %v", pod.Name, err)
```

Comment:
please revert it, if the pod is not found, it is considered successful.

Reply:
This is where I was confused, sure, I can revert but shouldn't it throw an error? Is this coming from k8s style?